### DISCUSSION TITLE
How to do a table of contents with NW's HTML format?

### DISCUSSION THREAD

**Discussion by AlanDunaway** [vkbo/novelWriter#1545] [Help]
Hello, I'd like to know.
How can we create a table of contents in the HTML file NovelWriter exports? For example, I'd like to have tags in chapters, when I click the link in the ToC, it would jump to the chapter I want.
Like:
> #! Book title
> Author
> Table of contents
> Chapter 1(link)
> Chapter 2(link)
> ...
> Chapter 1(jump to this if user clicks the link)

Would it be possible in the future?

**Comment by vkbo** [comment]
Yes, it's something I've considered. Please create a feature request for it, and I'll add it to the list of things to include in the build manuscript tool.

**Reply by vkbo** [reply]
Moved out of thread reply by @AlanDunaway (see #1393):

> Posted as issue #1548
